server/helpers/comments.js

```diff
@@ -18,15 +20,20 @@ export const comments = async (value) => {
         as: 'profile',
         attributes: ['bio', 'avatar']
       }]
-    }]
+    },
+    {
+      model: models.CommentLike,
+      attributes: []
+    }],
+    group: ['Comment.id', 'author.id', 'author->profile.id']
   });
 
   return payload;
 };
 
 export const singleComment = async (id) => {
   const payload = await Comment.findOne({
-    attributes: ['id', 'createdAt', 'updatedAt', 'body'],
+    attributes: ['id', 'createdAt', 'updatedAt', 'body', [Sequelize.fn('COUNT', Sequelize.col('CommentLikes.userId')), 'likeCount']],
```

Comment:
good job.

But if you notice, `Comment.findOne({` and `Comment.findAll({` have similar content (only difference is likely the `where` clause.

I would like to see you employ DRY principles here.

Reply:
I just implemented the feedback. Thanks